package.json

```diff
@@ -15,6 +15,7 @@
     "dommatrix": "^0.0.6",
     "es-module-shims": "^1.3.0",
     "escodegen": "^2.0.0",
```

Comment:
Shouldn't we remove the old `escodegen`? As far as I can tell the preprocessor is the only place that uses it, unless I missed something of course.

Reply:
My hope was that the original `escodegen` package will actually be updated (and actively maintained) in the future, such that we don't need to use the fork. Hence I figured that leaving this here would make it easier to spot when/if there's a new release.

If you don't think that's reason enough to keep this, and I can certainly understand that point of view, I can of course remove this package.